Eplant/views/ChromosomeViewer/Viewer/GeneAnnotation.tsx

```diff
@@ -0,0 +1,81 @@
+// -------
+// IMPORTS
+// -------
+import { FC } from 'react'
+
+import { useActiveGeneId } from '@eplant/state'
+import useTheme from '@mui/material/styles/useTheme'
+
+import { GeneAnnotationItem } from '../types'
+
+//----------
+// TYPES
+//----------
+interface GeneAnnotationProps {
+  gene: GeneAnnotationItem
+  scale: number
+}
+//----------
+// COMPONENT
+//----------
+const GeneAnnotation: FC<GeneAnnotationProps> = ({ gene, scale }) => {
+  const [activeGeneId, setActiveGeneId] = useActiveGeneId()
+  const theme = useTheme()
+
+  const onClick = () => {
+    setActiveGeneId(gene.id)
+  }
+  return (
+    <g id={`${gene.id}_annotation`} cursor='pointer'>
+      {/* GENE ID INDICATOR LINE */}
+      <line
+        x1={gene.strand == '+' ? 18 : 8}
+        y1={
```

Comment:
On my local there seems to be a slight offset in the y-direction of the gene annotation lines:

<img width="355" alt="image" src="https://github.com/user-attachments/assets/09b19ee8-a960-42c4-9702-f848e563f08c">

It seems to be zoom agnostic so might just require a small adjustment. Let me know if this is just on my end